src/parallel/LoadCalc.cpp

```diff
@@ -251,22 +251,27 @@ int MeasureLoad::prepareLoads(DomainDecompBase* decomp, MPI_Comm& comm) {
 	int global_maxParticlesP1 = 0;  // maxParticle Count + 1 = degrees of freedom
 	MPI_Allreduce(&maxParticlesP1, &global_maxParticlesP1, 1, MPI_INT, MPI_MAX, comm);
 
+#ifndef MARDYN_MEASURECALC_V1
```

Comment:
where is this defined? I could not find it throughout the repository.

Reply:
it's a sort-of-old macro that can be defined directly using the CXXFLAGS...
It provides a different version of the measureload calculation, but it had problems with stability resulting in negative loads.
I just ran this and I think that it can probably be removed.
An alternative would be a significant rewrite of it to also use an NNLS (non-negative least squares) algorithm, but that will still provide problems if the number of cells with a specific particle count is small...

Reply:
update: removed it + rename.